10/RequestReply.md

```diff
@@ -83,7 +83,37 @@ TODO understand the PUSH_BACK model. CORBA or more generic?
 
 #### Cache
 
-#### Serialisation
+#### Serialization
+
+The Serialization Mode defines how the Device Server handles multiple requests. 
+
+The Serialization SHALL apply to all requests independently of a request origin (network, the same machine, the same Device Server process). 
+
+A Device Server MAY handle requests with the following Serialization Modes:
+* Serialization by Device
+* Serialization by Class
+* Serialization by Process
+* No Serialization
+
+If Device Server is set to Serialization by Device:
+* it SHALL process requests to each Device in the order of requests arrival,
+* for any of Devices, it MUST NOT start the processing of a new request until the processing of the previous request is not finished,
```

Comment:
s/for any of Devices/for any Device in the Device Server/

Reply:
```suggestion
* for any Device in the Device Server, it MUST NOT start the processing of a new request until the processing of the previous request is not finished,
```